mirgecom/steppers.py

```diff
@@ -27,9 +27,13 @@
 THE SOFTWARE.
 """
 
+from logpyle import set_dt
+from mirgecom.logging_quantities import set_state
+from pyinstrument import Profiler
+
 
 def advance_state(rhs, timestepper, checkpoint, get_timestep,
-                  state, t_final, t=0.0, istep=0):
+                  state, t_final, t=0.0, istep=0, logmgr=None, instrumenter=None):
```

Comment:
I don't agree that integrating pyinstrument here is a good idea. Likely, you would want to profile over multiple time steps.

Reply:
I agree with your point, but from what I can see, pyinstrument is relatively inflexible, in that you can't break down the profiling results to a finer granularity after profiling.

Reply:
Removed pyinstrument.